code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -445,5 +445,140 @@
+		return
+	if(isloyal(target))
+		if(prob(20))	//loyalty implants typically overpower this, but you CAN get lucky and convert still (20% chance of success)
+			faith -= 125	//yes, this puts it negative. it's gonna take longer to recharge if you manage to convert a one of these people to balance the new power you gained through them
+			to_chat(missionary, "<span class='notice'>Through sheer willpower, you overcome their closed mind and rally [target] to your cause! You may need a bit longer than usual before your faith is fully recharged, and they won't remain loyal to you for long ...</span>")
+			convert_duration = 3000		//5 min, because the loyalty implant will attempt to counteract the subversion
+		else	//80% chance to fail
+			faith -= 75
+			to_chat(missionary, "<span class='warning'>Your faith is strong, but their mind remains closed to your ideals. Your resolve helps you retain a bit of faith though.</span>")
+			return
+	else if(target.mind.assigned_role == "Psychiatrist" || target.mind.assigned_role == "Librarian")		//fancy book lernin helps counter religion (day 0 job love, what madness!)
+		if(prob(35))	//35% chance to fail
+			to_chat(missionary, "<span class='warning'>This one is well trained in matters of the mind... They will not be swayed as easily as you thought...</span>")
+			faith -=50		//lose half your faith to the book-readers
+			return
+		else
+			to_chat(missionary, "<span class='notice'>You successfully convert [target] to your cause. The following grows because of your faith!</span>")
+			faith -= 100
+	else if(target.mind.assigned_role == "Civilian")
+		if(prob(55))	//55% chance to take LESS faith than normal, because civies are stupid and easily manipulated
+			to_chat(missionary, "<span class='notice'>Your message seems to resound well with [target]; converting them was much easier than expected.</span>")
+			faith -= 50
+		else		//45% chance to take the normal 100 faith cost
+			to_chat(missionary, "<span class='notice'>You successfully convert [target] to your cause. The following grows because of your faith!</span>")
+			faith -= 100
+	else		//everyone else takes 100 faith cost because they are normal
+		to_chat(missionary, "<span class='notice'>You successfully convert [target] to your cause. The following grows because of your faith!</span>")
+		faith -= 100
+	//if you made it this far: congratulations! you are now a religious zealot!
+	target.mind.make_zealot(missionary)
+
+	target << sound('sound/misc/wololo.ogg', 0, 1, 25)
+	missionary.say("WOLOLO!")
+	missionary << sound('sound/misc/wololo.ogg', 0, 1, 25)
+
+	var/obj/item/clothing/under/jumpsuit = null
+
+	if(target.w_uniform)
+		jumpsuit = target.w_uniform
+		jumpsuit.color = team_color
+		target.update_inv_w_uniform(0,0)
+
+	log_admin("[ckey(missionary.key)] has converted [ckey(target.key)] as a zealot.")
+	addtimer(src, "do_deconvert", convert_duration, FALSE, target, missionary, jumpsuit)	//deconverts after the timer expires
```

Comment:
fake sure they still properly deconvert if the staff is destroyed, otherwise, you'll have powergamers like me who will get a whole bunch of converts, then destroy the staff to have perma-followers.